cli/web_worker.rs

```diff
@@ -125,29 +131,29 @@ mod tests {
       })
     });
 
-    let r = tokio_util::run_basic(worker);
+    let mut rt = tokio_util::create_basic_runtime();
+    let r = run_worker_loop(&mut rt, &mut worker);
     assert!(r.is_ok())
   }
 
+  // FIXME(bartlomieju)
+  #[ignore]
```

Comment:
Are the changes to this test necessary? I'm fine with it getting disabled but maybe it shouldn't also change behavior.

Reply:
Yes; `onmessage = () => { delete self.onmessage; }` makes no sense. I'll try to fix those tests now

Reply:
Fixed